pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_what-is-a-module.adoc

```diff
@@ -1,4 +1,3 @@
-include::../attributes.adoc[]
```

Comment:
You have an attribute in the module: {ProductShortName}. If you're removing the attributes file, you need to hard code this. Why are you removing the attributes file reference?

Reply:
Hey @stoobie - modules must not contain includes, even to attributes files. In this case, the attributes shall be supplied at the assembly level, and there is also some ongoing discussion around a solution for standalone modules that we can talk about during the sprint review.

For the time being, this change is designed to bring the modules in line with what we have today, and when it comes time to implement the standalone module change, we'll do that as part of a broader update.